### PR TITLE
refactor(apport): Use logging library for logging

### PR DESCRIPTION
**Pull request by bdrung**
To make the code better readable, use the standard `logging` library for logging in `data/apport`.

### CODE REVIEW

File: data/apport [hunk 1]
```diff
@@ -179,22 +181,19 @@ def init_error_log():
         sys.stderr = io.TextIOWrapper(sys.stderr)
         sys.stdout = sys.stderr
 
-
-def error_log(msg):
-    """Output something to the error log."""
-
-    apport.error("apport (pid %s) %s: %s", os.getpid(), time.asctime(), msg)
+    logging.basicConfig(format=LOG_FORMAT)
```
Comment (schopin-pro) [inline data/apport:184]: question: the function could do an early return, should we take it into account?

**Review by schopin-pro — APPROVED** [review]
Overall LGTM but I have a slight concern around logger initialization.

File: data/apport (hunk 1 above)
Comment (bdrung) [reply data/apport]: Good catch. I moved the logging configuration into the main function.

**Review by schopin-pro — APPROVED** [review]

**Comment by bdrung** [comment]
Ignoring failed `codecov/patch` in this case.

**Pull request merged**